.circleci/config.yml

```diff
@@ -74,6 +74,18 @@ commands:
           webhook: $SLACK_GREENKEEPER_WEBHOOK
 
 jobs:
+  auto-rebase:
+    docker:
+      - image: boltdev/m2-installed-plugin-ci-php72:2.3.0-v2
+        auth:
+          username: $DOCKERHUB_USER
+          password: $DOCKERHUB_PASS
+    steps:
+      - checkout
+      - run:
+          name: Auto-rebase
+          command: bash .circleci/scripts/auto-rebase.sh
```

Comment:
Does this mark the commit as failed because of this? 
We may not want that? 
I am not blocking merge but if this is the case lets figure out how to clean up post merge

Reply:
jz/rebase-custom1 and jz/rebase-custom2 are real branches I created to test auto-rebasing. We can update them to customization branches once testing passes.